common/reviews/api/rush-lib.api.md

```diff
@@ -142,6 +152,20 @@ export interface IExperimentsJson {
     usePnpmFrozenLockfileForRushInstall?: boolean;
 }
 
+// @internal
+export interface _ILastInstallFlagJson {
```

Comment:
Why are we exporting this? If anyone needs it, they are probably doing something wrong

Reply:
Was simply exporting because the LastInstallFlag class was also exported, and this is part of the constructor. However, I've gone and removed the class from being exported as well. Let me know if this solution isn't something we want